front-end/src/app/shared/components/transaction-type-base/transaction-type-base.component.ts

```diff
@@ -372,6 +377,53 @@ export abstract class TransactionTypeBaseComponent implements OnInit, OnDestroy
     return transactionType?.generatePurposeDescription !== undefined;
   }
 
+  /**
+   *
+   * @param payload
+   *    Takes a SchATransaction because a Transaction does not have the
+   *    contributor_organization_name or contribution_purpose_description
+   *    fields.
+   * @returns
+   *    An array of Transaction objects whose contribution_purpose_descriptions
+   *    have been re-generated to account for changes to their parent
+   */
+  updateChildPurposeDescriptions(payload: SchATransaction): Transaction[] {
```

Comment:
I'm finding that this doesn't work when i change the name of the committee name of the parent transaction, because the user hasn't accepted the contact change yet at this point

Reply:
Disregard! this is working